scripts/nlp_language_modeling/preprocess_data_for_megatron.py

```diff
@@ -132,6 +136,12 @@ def get_args():
     group = parser.add_argument_group(title='runtime')
     group.add_argument('--workers', type=int, default=1, help='Number of worker processes to launch')
     group.add_argument('--log-interval', type=int, default=100, help='Interval between progress updates')
+    group.add_argument(
+        '--preproc-folder',
+        action='store_true',
+        help='If set, will preprocess all .json or .json.gz files into a single .bin and .idx file. Folder path provided via the --input arg',
```

Comment:
Can you add help text to the --input arg explaining that input can be a single file or folder?

Reply:
Good point! Done.